File: dotnet-back-end/ktdi-app/Model/Authentication.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

public class Authentication
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int UserId { get; set; }

    [Required]
    [EmailAddress]
    public string Email { get; set; }

    [Required]
    public string Password { get; set; }

    [Phone]
    public string PhoneNumber { get; set; }

    public bool OtpVerified { get; set; }

    public string Role { get; set; } = "user";
}

File: springboot-back-end/src/main/java/com/model/Authentication.java
package com.model;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "Authentication")
@Data
public class Authentication {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "user_id")
    private Long userId;

    @Column(unique = true, nullable = false)
    private String email;

    @Column(nullable = false)
    private String password;

    @Column(name = "phone_number")
    private String phoneNumber;

    @Column(name = "otp_verified")
    private Boolean otpVerified = false;

    private String role = "user";
}